backend/data_adapter_test.go

```diff
@@ -207,6 +207,24 @@ func TestQueryData(t *testing.T) {
 				expErrorSource:    ErrorSourcePlugin,
 				expError:          true,
 			},
+			{
+				name: `single error that we override as "downstream" should be "downstream" error source even if "plugin" error source`,
```

Comment:
I am a little bit fearful over the idea of overriding what up until this point has been the source of truth. Would like to have @marefr's opinion on this before we merge 🙏

Reply:
Okay, maybe if we don't want to do that, we could just skip logging cancelled errors
```
if r.Error == nil || RequestStatusFromError(r.Error) == RequestStatusCancelled {
				continue
			}
```

Reply:
I answered here https://github.com/grafana/grafana-plugin-sdk-go/pull/1097/files#r1781017045. 

I think you should focus on the plugin request completed and use partial data response error for further details if needed. Not sure how you use these two in combination right now